x-pack/plugins/fleet/public/components/agent_enrollment_flyout/download_instructions.tsx

```diff
@@ -0,0 +1,187 @@
+/*
```

Comment:
I extracted this component from the `DownloadStep` to avoid any rendering issue. Since the `steps` components are not actual react components but return `EuiContainedStepProps` instead, if any hook is used inside them, they  tend to break badly. This solution avoids getting into that state.